### ISSUE TITLE
[P2] Repair the obsolete training E2E test and add reproducible numerical and lifecycle regressions

### ISSUE THREAD

**Issue by KadoBOT** [KadoBOT/ComfyUI-YuE2-Voices#15]
## Scope
Testing/reproducibility audit at `29c0e939261067afae1ad517f23aab50d62648db`. The obsolete call signature and missing checks are visible statically; the full repository test suite and GPU tests were not run in this audit.

## Definite test breakage
[`e2e_train_node.py`](https://github.com/KadoBOT/ComfyUI-YuE2-Voices/blob/29c0e939261067afae1ad517f23aab50d62648db/e2e_train_node.py) still invokes the removed audio/lyrics/start/end positional interface:
```python
kwargs = dict(style='test style', yue2_checkpoint=..., ...)
YuE2VoiceTrain.execute(track_audio, lyrics, 5.0, 20.0, **kwargs)
```
The current [`YuE2VoiceTrain.execute`](https://github.com/KadoBOT/ComfyUI-YuE2-Voices/blob/29c0e939261067afae1ad517f23aab50d62648db/nodes.py) begins with checkpoint/tokenizer arguments and accepts `track1_audio`, `track1_style`, and `track1_lyrics`; it has no `style` keyword or those range arguments. With prerequisites available, argument binding fails before training, including duplicate checkpoint assignment. The test therefore cannot validate the current public path.

## Coverage and reproducibility gaps
- [`worker/verify_backbone.py`](https://github.com/KadoBOT/ComfyUI-YuE2-Voices/blob/29c0e939261067afae1ad517f23aab50d62648db/worker/verify_backbone.py) compares unadapted AR logits. That is useful but does not prove adapted AR KV compatibility or exported NAR velocity equivalence.
- [`e2e_apply_node.py`](https://github.com/KadoBOT/ComfyUI-YuE2-Voices/blob/29c0e939261067afae1ad517f23aab50d62648db/e2e_apply_node.py) checks changed AR weights/counts using a hardcoded private-local smoke voice. It does not test the NAR exporter/loader or the full saved-vocalist lifecycle.
- [`test_nodes.py`](https://github.com/KadoBOT/ComfyUI-YuE2-Voices/blob/29c0e939261067afae1ad517f23aab50d62648db/test_nodes.py) has useful CPU audio/metadata coverage, but its retraining tests check that file changes alter fingerprints—not whether training's own output triggers another unchanged queue. Its checksum test covers a new download, not an existing same-size corrupt file.
- [`checks/nar_train.py`](https://github.com/KadoBOT/ComfyUI-YuE2-Voices/blob/29c0e939261067afae1ad517f23aab50d62648db/checks/nar_train.py) seeds Python and torch but samples flow timesteps with unseeded `np.random.beta`. Nominally identical runs do not have the same timestep sequence. Several scripts also use shared append logs/output paths and unconditional top-level execution, making import tests and run provenance fragile.

## Acceptance criteria
- [ ] Update E2E calls to the actual named-input contract; use synthetic or explicitly supplied fixtures and fail/skip clearly when model prerequisites are absent.
- [ ] CPU CI covers raw/absolute token IDs, sequence bounds and late-track coverage, short-clip shapes, sample-rate preservation, same-size corruption, export discovery, argument binding, and deliberate retraining idempotence.
- [ ] GPU integration compares adapted AR logits and per-layer KV states, plus original-versus-exported NAR predictions at identical latents/timesteps/conditioning. Cover merged projections, alpha/scaling, and I/O weight/bias deltas.
- [ ] A frontend round trip imports the GUI graph and exports the effective API prompt, rather than validating only the builder's own parallel arrays.
- [ ] End-to-end smoke includes create candidate -> validate -> publish -> restart/reload -> generate, plus a second independent singer.
- [ ] Seed every RNG used, save their states when claiming resumability, and record fresh run IDs, resolved configuration, source revisions, and hashes. Exact cross-device determinism should not be promised without evidence.
- [ ] Make experiment modules import-safe, keep logs/artifacts per immutable run, and separate tests of numerical compatibility from tests of perceptual identity.
